cvat-ui/src/reducers/import-reducer.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import { ImportActions, ImportActionTypes } from 'actions/import-actions';
+import deepCopy from 'utils/deep-copy';
+
+import { ImportState } from './interfaces';
+
+const defaultState: ImportState = {
+    projects: {},
+    progress: 0.0,
+    status: '',
```

Comment:
Several activities, but one progress and status looks strange

Reply:
Maybe instance is enough?

Reply:
Added format value to track active import proccess